dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResourceHelper.java

```diff
@@ -190,4 +212,79 @@ public String asJson(final PageView pageView) throws JsonProcessingException {
         return objectWriter.writeValueAsString(pageView);
     }
 
+    public Template saveTemplate(final User user, String pageId, final PageForm pageForm)
+            throws BadRequestException, DotDataException, DotSecurityException, IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotcms.rest.exception.BadRequestException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)